content/hardware/04.pro/boards/portenta-x8/tutorials/02.x8-fundamentals/portenta-x8-fundamentals.md

```diff
@@ -1,85 +1,233 @@
+- Enhanced cybersecurity features like **OP-TEE** and **OSTREE**.
+- **Over-The-Air (OTA)** system updates.
+- Integration with **Docker Compose** to deploy entire containers for applications.
 
-*Foundries.io* provides a service that builds images using the Yocto Project and is specifically built around the Linux microPlatform (LmP) distribution they maintain. LmP contains an extensive set of software components needed for IoT applications.
+Foundries.io also offers **FoundriesFactory**, a Cloud-based *DevSecOps* subscription service. It allows you to build, test, deploy, and maintain secure IoT and edge devices. Each Factory has a unique ID and automatic builds of both the base system and containers are handled in one place, streamlining the development process and ensuring a secure, up-to-date environment for your IoT and Edge devices.
 
-Using [fioctl](https://docs.foundries.io/latest/getting-started/install-fioctl/index.html) allows you to manage your boards through CLI. It makes it easy to upload containers to a board linked to your Factory. When the board is online and connected to the Factory, you can easily push new apps to the board. Using the fioctl command lines, you only need to state the Factory, board, and app.
+#### Factory Repositories
+
+The Factory uses four main repositories to customize images:
+
+- **ci-scripts.git**: Defines build jobs for the continuous integration system.
+- **lmp-manifest.git**: Specifies the platform’s image layers.
+- **meta-subscriber-overrides.git**: Customizes the Factory image by adding or removing packages and implementing board-specific features.
+- **containers.git**: Defines containers and how they should be orchestrated using Docker or `docker compose`.
+
+Foundries.io simplifies container management by building images with the Yocto Project and **Linux microPlatform (LmP)**, optimized for IoT.
+
+#### Managing Containers in the Factory
+
+Through the Factory, you can monitor connected devices, track the containers running on them, and manage different versions of these containers. Each time a repository is updated, the continuous integration system generates a new target that can be deployed on your Portenta X8 devices.
+
+Foundries.io simplifies container management, building images using the Yocto Project and their **Linux microPlatform (LmP)** distribution, which is optimized for IoT and embedded systems. LmP contains an extensive set of components designed for IoT applications, allowing for fast, consistent deployments across different environments.
+
+#### Managing Devices with **fioctl**
+
+The Foundries.io tool *fioctl* enables command-line management of your devices, allowing you to upload containers, push updates, and deploy new applications seamlessly. It simplifies the deployment process across multiple devices, ensuring smooth integration and management of containers within your Factory.
+
+#### Docker Containers in Action

```

Comment:
Please move this to a Containers section and separate it better from foundries.io... Otherwise people might think that you need foundries to use containers, even if that is not the case

Reply:
Updated!